llm/gguf.go

```diff
@@ -225,31 +227,31 @@ func (llm *gguf) Decode(rs io.ReadSeeker) error {
 	// patch KV with parameter count
 	llm.kv["general.parameter_count"] = llm.parameters
 
-	alignment, ok := llm.kv["general.alignment"].(uint32)
-	if !ok {
-		alignment = 32
-	}
+	// alignment, ok := llm.kv["general.alignment"].(uint32)
```

Comment:
We're good to remove support for concatenated gguf files

Reply:
Update: Not yet. We left it in an fixed some bugs in tests with it.